pkgs/development/libraries/hfst-ospell/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, autoreconfHook
+, fetchFromGitHub
+, icu
+, libarchive
+, libxmlxx
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hfst-ospell";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "hfst";
+    repo = "hfst-ospell";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-16H1nbAIe+G71+TnlLG0WnH9LktZwmc0d0O+oYduH1k=";
+  };
+
+  buildInputs = [
+    icu
+    libarchive
+    libxmlxx
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+  ];
+
```

```suggestion
  configureFlags = [
    "--without-libxmlpp"
    "--without-tinyxml2"
  ];
```
This fixes build on darwin. See https://github.com/hfst/hfst-ospell/issues/48#issuecomment-546535653

---

Maybe we should link the issue as a comment